canonical-web-and-design/ubuntu.com
# Rebuilds /openstack/consulting

## Done

- Rebuilds /oenstack/consulting based on copydoc changes

**NOTE: The pie-chart is being built in a separate PR**

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/consulting
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copydoc](https://docs.google.com/document/d/1f83OSIHVQGMjF7b0pZUKFvoubMGqi_b-OYbSpyhveX4/edit#heading=h.e1gnvwq12nci)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4386
